packages/shared/world/TeleportController.ts

```diff
@@ -78,6 +101,19 @@ export class TeleportController {
   }
 }
 
+async function fetchHomePoint(): Promise<{ x: number; y: number }> {
+  const homePoint: string = await getFromPersistentStorage(homePointKey)
+  if (homePoint) {
+    store.dispatch(sendHomeScene(homePoint))
+    if (!HAS_INITIAL_POSITION_MARK) {
+      const [x, y] = homePoint.split(',').map((p) => parseFloat(p))
+      gridToWorld(x, y, lastPlayerPosition)
+      return { x, y }
+    }
```

Comment:
Is this condition right? why does it has a HAS_INITIAL_POSITION_MARK if?

Reply:
In my ignorance, I thought it was needed, sorry.